terraform/cluster/aws/main.tf

```diff
@@ -46,7 +46,16 @@ resource "null_resource" "wait_k8s_api" {
   depends_on = [
     aws_eks_cluster.cluster
   ]
+}
+
+resource "null_resource" "wait_k8s_cluster" {
+  provisioner "local-exec" {
+    command = "sleep 10"
+  }
 
+  depends_on = [
```

Comment:
should this be `aws_eks_node_group.cluster` ?

Reply:
thanks for pointing it out, I've fixed the other providers